### PR TITLE
Documentation for searching types, orderby, and added search operators examples.

### PR DESCRIPTION
**Pull request by sukhdev1103**

### CODE REVIEW

**Comment by sukhdev1103** [comment]
Had to commit so many times due to table format :(. It was not coming properly.

File: Documentation/SequentialDataStore/Searching.md [hunk 1]
```diff
@@ -4,96 +4,305 @@ uid: sdsSearching
 
 #### Qi_Searching_topic
 
+Searching
+=====================
+SdsSearch provides a way to search text, phrases, fields etc over the Sequential Data Store. This document covers the
```
Comment (sukhdev1103) [inline Documentation/SequentialDataStore/Searching.md]: and,

Comment (sukhdev1103) [reply Documentation/SequentialDataStore/Searching.md]: provides a way to search text, phrases, and fields, etc.,

Comment (sukhdev1103) [reply Documentation/SequentialDataStore/Searching.md]: cross the Sequential Data Store.

File: Documentation/SequentialDataStore/Searching.md [hunk 2]
```diff
@@ -4,96 +4,305 @@ uid: sdsSearching
...
 
 
-The ``GetStreamsAsync`` overload returns streams that match specific search criteria within a given namespace. 
-You use the ``searchText`` parameter to specify a search string. The ``GetStreamsAsync`` method then returns any 
-stream that matches the search string in the ``name``, ``description``, or ``tag`` list. 
+Searching for types
+=====================
+
+Similarly, the search functionality for types is also exposed through REST API and the client libraries method ``GetTypesAsync``. The query syntax and the request parameters are the same. 
+The only difference is the resource you're searching on, and the type properties that can be filtered on are different than on streams. See [Types](xref:sdsTypes) for more information.
```
Comment (sukhdev1103) [inline Documentation/SequentialDataStore/Searching.md]: You can filter on different properties for types than for streams.

File: Documentation/SequentialDataStore/Searching.md (hunk 1 above)
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Searching.md]: etc.

File: Documentation/SequentialDataStore/Searching.md [hunk 3]
```diff
@@ -4,96 +4,305 @@ uid: sdsSearching
...
-stream2      | pressureA | The pressure     | “pressure”, “DeviceA”
-             |           | from DeviceA     |
-stream3      | calcA     | calculation from | “temperature”, 
-             |           | DeviceA values   | “pressure”, “DeviceA”
+stream1      | tempA     | The temperature from DeviceA | “temperature”, “DeviceA”
+stream2      | pressureA | The pressure from DeviceA    | “pressure”, “DeviceA”
+stream3      | calcA     | calculation from DeviceA values | “temperature”, “pressure”, “DeviceA”
 
 
 Using the stream data above, the following table shows the results of a ``GetStreamsAsync`` call with different ``SearchText`` values:
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Searching.md]: QueryString

**Pull request merged**